### PR TITLE
add new examples

### PR DESCRIPTION
**Pull request by bcdunbar**
Adds:
- https://github.com/plotly/dash-vanguard-report to the gallery
- https://github.com/plotly/dash-opioid-epidemic-demo to the gallery
- screenshots/gifs to /images

### CODE REVIEW

File: tutorial/gallery.py [hunk 1]
```diff
@@ -291,6 +291,46 @@
 
     ***
 
+    ## Opioid Epidemic
+
+    Interactively explore the affect of the opioid epidemic in North America.
```
Comment (chriddyp) [inline tutorial/gallery.py]: *`effect`

File: tutorial/gallery.py [hunk 2]
```diff
@@ -291,6 +291,46 @@
...
+    ),
+    dcc.Markdown('''
+
+    [View the app](https://opioid-epidemic.herokuapp.com/) | [View the source code](https://github.com/plotly/dash-opioid-epidemic-demo)
+
+    ***
+
+    ## Vanguard Report
+
+    This app illustrates the ability to create interactive reports with Dash. View, interact and then print the report in either single page or full view.
```
Comment (chriddyp) [inline tutorial/gallery.py]: Let's use the copy that we had for the previous report here: https://github.com/plotly/dash-docs/commit/1ee74ec5b60da5d5948d6d64b39a3ed5e2137dbe#diff-37a075e07f30088115d976c468026e82L72

Comment (chriddyp) [reply tutorial/gallery.py]: Also, let's try to keep the line length short - 80 characters or less.

**Comment by chriddyp** [comment]
Thanks @bcdunbar ! Tag me when you've made those two changes and I'll review again

**Comment by bcdunbar** [comment]
@chriddyp or if too busy someone else @plotly/dash

**Comment by chriddyp** [comment]
💃  - Thanks! Merging will automatically deploy the app, so make sure to check the docs 5-10 minutes after you deploy. I'll leave it to you!

**Pull request merged**